index.js

```diff
@@ -523,15 +528,6 @@ var compile = function(schema, cache, root, reporter, opts) {
   validate = validate.toFunction(scope)
   validate.errors = null
 
-  validate.__defineGetter__('error', function() {
```

Comment:
I haven't removed the validate.errors feature, just implemented in a different way. __defineGetter__ is non standard and has been deprecated. I can still add it back if you like?